decidim-meetings/app/views/decidim/meetings/meetings/new.html.erb

```diff
@@ -1,25 +1,21 @@
-<div class="row">
-  <div class="columns large-3">
-    <%= link_to :back do %>
-      <%= icon "chevron-left", class: "icon--small", role: "img", "aria-hidden": true %>
-      <%= t(".back") %>
-    <% end %>
+<%= render layout: "layouts/decidim/shared/layout_center" do %>
+  <div class="text-center py-10">
+    <h1 class="h3 md:h1 decorator inline-block text-left">
+      <%= t("title", scope: "decidim.meetings.meetings.new") %>
+    </h1>
   </div>
 
-  <div class="columns large-6">
-    <h2 class="section-heading"><%= t(".title") %></h2>
-    <div class="card">
-      <div class="card__content">
-        <%= decidim_form_for(@form, html: { class: "meetings_form new_meeting" }) do |f| %>
-          <%= render partial: "form", object: f %>
-
-          <div class="actions">
-            <%= f.submit t(".create"), class: "button expanded", data: { disable: true } %>
-          </div>
-        <% end %>
-      </div>
+  <%= decidim_form_for(@form, html: { class: "meetings_form new_meeting" }) do |f| %>
+    <div class="form__wrapper">
+      <%= render partial: "form", object: f %>
```

Comment:
This is not a problem **only** at meetings but at all forms. The fields show that "there is an error" when I enter the new meeting page:

![Errors on form](https://user-images.githubusercontent.com/864340/209187805-167df06d-5dbe-4037-9e68-4e3f58500be8.png)

It should only show the errors after a user action, not directly on the page. This is confusing for the user and is an accessibility violation.

This particular issue applies to most forms.

Reply:
I've created [a separate issue](https://github.com/decidim/decidim/issues/10191). Originaly was in https://github.com/decidim/decidim/issues/9674